### DISCUSSION TITLE
Where can I read the function definition of resetJointStatesMultiDof?

### DISCUSSION THREAD

**Discussion by HaziqRazali** [bulletphysics/bullet3#3418] [General]
Where can I read the function definition of resetJointStatesMultiDof? I have been looking through the GitHub page but can't find it. I don't want to know how to use it, rather, I would like to go through its code.

**Comment by rohit-kumar-j** [comment]
On a similar note, I was looking for the function definition of pybullet.getQuaternionSlerp().  #3164 Is there a way to read the pre-existing function definitions within pybullet.
